### PR TITLE
Reduce scope and update the dates

### PR DESCRIPTION
**Pull request by klizhentas**

### CODE REVIEW

File: docs/5.0/preview/upcoming-releases.md [hunk 1]
```diff
@@ -30,9 +30,15 @@ and Access Workflows UI.
...
 |----------------------|-------------------|---------------------------
 | First alpha          | Jan 15th, 2021    | Good for testing and demos.
-| First beta           | Feb 1st, 2021     | Deploy on staging.
+| First beta           | Feb 15th, 2021    | Deploy on staging.
 | Release              | March 1st, 2021   | Good to go for production.
 
+!!! warning
+
+    We pushed beta release date to Feb 15th 2021.
+    We moved U2F and Access workflows to 6.1 release to reduce scope and match the promised
```
Comment (r0mant) [inline docs/5.0/preview/upcoming-releases.md:39]: Nit: Access **W**orkflows.

**Review by r0mant — APPROVED** [review]

File: docs/5.0/preview/upcoming-releases.md [hunk 2]
```diff
@@ -30,9 +30,15 @@ and Access Workflows UI.
 | Version              | Date              | Description
 |----------------------|-------------------|---------------------------
 | First alpha          | Jan 15th, 2021    | Good for testing and demos.
-| First beta           | Feb 1st, 2021     | Deploy on staging.
+| First beta           | Feb 15th, 2021    | Deploy on staging.
```
Comment (russjones) [inline docs/5.0/preview/upcoming-releases.md:33]: Line 24 needs to be updated to say:

```
The team open sources role based access controls and implements Database Access.
```

File: docs/5.0/preview/upcoming-releases.md [hunk 3]
```diff
@@ -43,10 +49,30 @@ You can find the full list of fixes and features in the
 | Database Access                     | All               | SSO into PostgreSQL or MySQL. Use AWS Aurora RDS or on-premises. Read [more here](./database-access.md).
 | OSS RBAC                            | OSS               | Open source role based access controls. Check out the [design doc](https://github.com/gravitational/teleport/blob/master/rfd/0007-rbac-oss.md) and [issue 4136](https://github.com/gravitational/teleport/issues/4136).
 | Terraform Provider                  | All               | Configure Teleport without UI using Terraform provider. More details [here](https://github.com/gravitational/teleport-plugins/projects/3#card-49866475).
+| Client libraries and API            | All               | Use Go to create access workflows. Review the [design doc](https://github.com/gravitational/teleport/pull/4746) and [issue 4763](https://github.com/gravitational/teleport/issues/4763).
```
Comment (russjones) [inline docs/5.0/preview/upcoming-releases.md:52]: For the Client libraries and API work, the following is what we're on track for.

6.0

1. Auth GRPC API moved to `api` package with limited dependencies.
2. Additional CRUD endpoints needed for Terraform provider.
3. Support connecting to Auth through Proxy (needed for Cloud).
4. Credential loader.

6.1

1. Move Access requests code into client from `teleport-plugins`.
2. Provide simplified high level access hooks.

Comment (klizhentas) [reply docs/5.0/preview/upcoming-releases.md:50]: @russjones can you remove things that we are not going to deliver from 4673 and create a new issue in 6.1 with extra deliverables

Comment (russjones) [reply docs/5.0/preview/upcoming-releases.md:50]: 6.0

https://github.com/gravitational/teleport/issues/5464

6.1

https://github.com/gravitational/teleport/issues/5465
https://github.com/gravitational/teleport/issues/5466

**Pull request merged**